desktop/core/src/desktop/js/apps/admin/AdminHeader.tsx

```diff
@@ -0,0 +1,77 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// 'License'); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react';
+import { Select, Input } from 'antd';
+import { SearchOutlined } from '@ant-design/icons';
+import './AdminHeader.scss';
+
+const { Option } = Select;
+
+interface AdminHeaderProps {
+  options: string[];
+  selectedValue: string;
+  onSelectChange: (value: string) => void;
+  filterValue: string;
+  onFilterChange: (value: string) => void;
+  placeholder: string;
+  configAddress?: string;
+}
+
+const AdminHeader: React.FC<AdminHeaderProps> = ({
+  options,
+  selectedValue,
+  onSelectChange,
+  filterValue,
+  onFilterChange,
+  placeholder,
+  configAddress
+}) => {
+  return (
+    <div className="cuix antd admin-header-actions">
+      <Select
+        value={selectedValue}
+        onChange={value => onSelectChange(value)}
```

Comment:
We want to pass value here only and not the entire object because of antd. Also its hard to pass the unit test without passing value.

Reply:
But you are not adding or modifying anything by wrapping onSelectChange in this function, the result is exactly the same

Reply:
We explicitly pass a specific value to onSelectChange. This makes it easier to test because we can mock or spy on onSelectChange and verify it was called with the correct argument while avoiding mocking the event object. Mocking was becoming difficult because we had to mock the whole event object.